### ISSUE TITLE
HTML export is not available even 'jinja' extra requirement is installed

### ISSUE THREAD

**Issue by sveetch** [sveetch/py-html-checker#23]
With recent new install even with the jinja extra requirement enabled, the "html" choice for --exporter argument is not available.

From tests we can see there is a problem with Jinja dependancy "MarkupSafe":

```
ImportError while importing test module '/home/thenonda/Applications/py-html-checker/tests/034_export_jinja.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
.venv/lib/python3.8/site-packages/_pytest/python.py:608: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
.venv/lib/python3.8/site-packages/_pytest/pathlib.py:533: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
.venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
tests/034_export_jinja.py:5: in <module>
    from jinja2 import Environment, Template
.venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
.venv/lib/python3.8/site-packages/jinja2/__init__.py:12: in <module>
    from .environment import Environment
.venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
.venv/lib/python3.8/site-packages/jinja2/environment.py:25: in <module>
    from .defaults import BLOCK_END_STRING
.venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
.venv/lib/python3.8/site-packages/jinja2/defaults.py:3: in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
.venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
.venv/lib/python3.8/site-packages/jinja2/filters.py:13: in <module>
    from markupsafe import soft_unicode
E   ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/home/thenonda/Applications/py-html-checker/.venv/lib/python3.8/site-packages/markupsafe/__init__.py)

```
It seems recent MarkupSafe have dropped the soft_unicode function which Jinja rely on.

**Comment by sveetch** [comment]
We will pin MarkupSafe to 2.0.1 at least temporarily. However it could be nicer to evolve to last supported Jinja 3.x to avoid managing its 2.x bugs.

**Comment by sveetch** [comment]
Fixed with pinned MarkupSafe==2.0.1 in 0.4.2 release.
